content/docs/scaffolder/scaffolder-actions-directory/index.md

```diff
@@ -5,18 +5,166 @@ description: Directory of available scaffolder actions
+| Sentry               | [sentry:create-project](#sentrycreateproject)      | Action to create a new project in Sentry.                                                                                                                                 | [Sentry Project Create](https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts)                                                                                                                                     | organizationSlug*, teamSlug*, name*, slug, authToken                                                    |
+| Torque               | [torque:create-app](torquecreateapp)            | Action for managing applications in Torque.                                                                                                                               | [Torque App Create](https://github.com/QualiTorque/torque-backstage-plugin/blob/main/packages/torque-backend/src/actions/create-app.ts)                                                                                                                                               | serviceName*, assetRepo*, blueprintRepo*, namespace*, blueprintName*, serviceAccount, repoType           |
+| Webex                | [webex:webhooks:sendMessage](#webexwebhookssendmessage) | Action that sends messages to Webex Incoming Webhooks.                                                                                                                    | [Webex SendMessage](https://github.com/Coderrob/backstage-plugin-scaffolder-backend-module-webex/blob/main/src/actions/sendWebhooksMessageAction.ts)                                                                                                                                  | format, message*, webhooks*                                                                             |
+| Yeoman               | [run:yeoman](#runyeoman)                  | Action for running a Yeoman generator in Backstage scaffolder.                                                                                                            | [Yeoman Run](https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-yeoman)                                                                                                                                                                             | namespace*, args, options                                                                               |
+
+## Download content
```

Comment:
This title doesn't seem like it fits here

Reply:
What category would you call these then?

Reply:
It's not the category. There is a full title missing. Now it's abruptly changing into a different list after the table.

Reply:
Is that better now?

Reply:
yup, looks good :+1: